package.json

```diff
@@ -21,7 +21,7 @@
     "record-build-info": "node ./bin/record-build-info",
     "lint": "eslint . --cache --max-warnings 0",
     "typecheck": "tsc && tsc -p integration_tests",
-    "test": "jest --maxWorkers=2",
+    "test": "TZ=Europe/London jest --maxWorkers=2",
```

Comment:
Need to replicated that we always run in London time to prove ISO data conversion works when in BST